packages/select/docs/Select/OptionWithSubLabel.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { HTMLAttributes } from "react"
```

Comment:
This would be done on a consumer level - here for example and testing purposes only